src/demo.js

```diff
@@ -17,7 +17,7 @@ class StatefulContainer extends React.Component {
                 </select>
             </p>
 
-            <HotelsOnMap ref='hotelsOnMap' hotels={this.hotels()}/>
+            <HotelsOnMap ref='hotelsOnMap' hotels={this.hotels()} mapMarkerImgUrl='' />
```

Comment:
Shouldn't the attribute be better omitted when the default value is good enough?